presto-native-execution/presto_cpp/main/PrestoTask.cpp

```diff
@@ -129,6 +129,71 @@ static void addRuntimeMetricIfNotZero(
   }
 }
 
+// Add 'running time' metrics from Velox operator stats to Presto operator
+// stats.
+static void addRunningTimeOperatorMetrics(
+    protocol::OperatorStats& opOut,
+    protocol::TaskStats& prestoTaskStats,
+    const exec::OperatorStats& op) {
+  auto statName = fmt::format(
+      "{}.{}.runningAddInputWallNanos", op.operatorType, op.planNodeId);
+  auto prestoMetric =
+      createProtocolRuntimeMetric(statName, op.addInputTiming.wallNanos);
+  opOut.runtimeStats.emplace(statName, prestoMetric);
```

Comment:
I didn't know that. Good to know. :)
I don't use the "Operator Performance" tab of the Presto Lookup UI much at all, only the "Runtime Stats" tab.
So I don't know if these new ones would be useful in "Operator Performance" tab.
What do you think?

Reply:
I feel it's better if we only add our runningAddInput() metrics to the "Runtime Stats" tab, but not to "Operator Performance" tab. It can keep the stats on "Operator Performance" tab short and only the operator unique stats there. We usually do x-operator comparison in the "Runtime Stats" tab as you do.

Reply:
Synced offline and decided to deal with this separately, because right now we add ALL runtime metrics into both op and task lists.